<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Registrar paquete</string>
     </attribute>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el nombre del objeto que va en el paquete...</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="cuantitySpinBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>42</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>64</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese la cantidad de objetos en el paquete</string>
      </property>
     </widget>
     <widget class="QLabel" name="resultLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>95</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Mis paquetes</string>
     </attribute>
     <widget class="QTableWidget" name="paquetesTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>741</width>
        <height>461</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>cedula</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>estado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>fecha de envío</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nombre de los productos</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>cantidad de productos</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refrescar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Buscar paquete</string>
     </attribute>
     <widget class="QLineEdit" name="idBuscarEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el id del paquete a buscar...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buscarButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>buscar</string>
      </property>
     </widget>
     <widget class="QLabel" name="resultadoLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>691</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Solicitar eliminación de paquete</string>
     </attribute>
     <widget class="QLineEdit" name="idEliminarEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ingrese el paquete a solicitar la eliminación...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="requestButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>solicitar</string>
      </property>
     </widget>
     <widget class="QLabel" name="messageLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>15</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
